CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 ## v1.3.0 (WIP)
 
+- Removed field ProviderID from Token
```

Comment:
This changelog note seems unrelated

Reply:
Updated with relevant changes for this PR.